enterprise/admin/__init__.py

```diff
@@ -597,6 +600,12 @@ class Meta:
     change_list_template = "enterprise/admin/enterprise_course_enrollments_list.html"
     search_fields = ('enterprise_customer_user__user_id', 'course_id',)
 
+    def license_uuid(self, obj):
+        """
+        Return the subscription license UUID (if any exists)  associated with this enrollment.
+        """
+        return str(obj.license.license_uuid)
```

Comment:
You checked this doesn't break on enrollments that don't have a license right?

Reply:
Yup, `license` is a property that safely returns  `None` if no associated `LicensedEnterpriseCourseEnrollment` exists.  But good general point that this could use a test.

Reply:
I'm gonna punt on this for now in the interest of time, and since there's no existing tests for most of the base ModelAdmin definitions.